tests/Lykke.Job.BlockchainCashinDetector.IntegrationTests/Utils/LaunchSettingsFixture.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Lykke.Job.BlockchainCashinDetector.IntegrationTests.Utils
+{
+    public class LaunchSettingsFixture : IDisposable
+    {
+        public LaunchSettingsFixture()
+        {
+            try
+            {
+                using (var file = File.OpenText("Properties\\launchSettings.json"))
```

Comment:
Why you accessing this file directly? Use ConfigurationBuilder instead

Reply:
How it should be implemented?
Pls, read this article. This is how to set xunit test env variables. It will not break on TC build + convenient to run locally. 
https://stackoverflow.com/questions/43927955/should-getenvironmentvariable-work-in-xunit-test

Reply:
+ it is not yet on Lykke.Sdk so appsettings.tests.json will not work